.gitops/charts/traffic-court-online/charts/workflow-service/templates/deployment.yaml

```diff
@@ -44,10 +44,8 @@ spec:
           {{- end }}
           - name: TZ
             value: America/Vancouver
-          - name: OracleDataApi__Host
-            value: {{ .Release.Name }}-oracle-data-api.{{ .Values.global.namespace }}.svc.cluster.local
-          - name: OracleDataApi__Port
-            value: "8080"
+          - name: OracleDataApi__BaseUrl
+            value: http://{{ .Release.Name }}-oracle-data-api.{{ .Values.global.namespace }}.svc.cluster.local:8080
```

Comment:
The route you configured in OpenShift is https

Reply:
you will notice this is not a public route, 

```
http://traffic-court-online-oracle-data-api.0198bb-dev.svc.cluster.local
```
and
```
http://traffic-court-online-oracle-data-api.0198bb-test.svc.cluster.local
```